test/core/models/ocelot/ProcessSpec.scala

```diff
@@ -102,4 +102,33 @@ class ProcessSpec extends BaseSpec with ProcessJson {
     }
   }
 
+  "Ocelot Backlink Behaviour" must {
+
+    "Be disabled by default" in {
+      Process(meta, flow, phrases, links).ocelotBacklinkBehaviour shouldBe false
+    }
+
+    "Be enabled when process contains a Value named _GuidanceBacklinkBehaviour and set to ocelot" in {
+      //case class Process(meta: Meta, flow: Map[String, Stanza], phrases: Vector[Phrase], links: Vector[Link], timescales: Map[String, Int] = Map())
+      val newValueStanza = flow(blankValueStanzaID) match {
+        case v: ValueStanza => v.copy(values = List(Value(ScalarType, s"${Process.BacklinkBehaviourLabelName}", "Ocelot")))
```

Comment:
We could add a few more tests to ensure different upper / lower case letter combinations for "ocelot" are accepted (like "oCelot", "oCElot").